quiz.js

```diff
@@ -0,0 +1,64 @@
+"use strict"
+
+	CarLot = (function(oldCarLot){
+
+    oldCarLot.displayCars = function(inventory){
+        var inCars = document.getElementsByClassName("row")[0];
+        var seeTheCars="";
+        for (var i = 0; i < inventory.length; i++) {
+            var currentCar = inventory[i];
+            seeTheCars += `<div class="carCard col-xs-4">
+            <p>${currentCar.year} ${currentCar.make} ${currentCar.model}</p>
+            <p>Price: ${currentCar.price}</p> Color:  ${currentCar.color} 
+            <p>Availability: ${currentCar.purchased}</p>
+            <p class="oldCopy">Description: ${currentCar.description}</p>
+            </div></div>`
+
+            inCars.innerHTML = seeTheCars;
+        };
+
+        		oldCarLot.activateEvents = function (carCards) {
+			for (var i = 0; i < carCards.length; i++) {
+				carCards[i].addEventListener("click", function(e){
+					let clickTarget = e.target.closest("div");
+                    selectCopy(e);  
+
+				})
+			}
+
+            var copyEntry = document.getElementById("copyChange");
+            var newCopy = document.getElementsByClassName("oldcopy");
+            var originalCard = document.getElementsByClassName("carCard");
+            var replaceCopy;
+
+
+            function selectCopy (event) {
+                replaceCopy = event.target.closest(".carCard").childNodes[6].nextElementSibling;
```

Comment:
Careful when you assign a number to a child or node. There may come a time when you want to change the way your HTML looks, and it will break your app. Instead try to focus on an id or class.